stats/stats.go

```diff
@@ -36,7 +36,49 @@ type RPCStats interface {
 	IsClient() bool
 }
 
+// InHeader contain stats when the header is received.
+//  First event in the server side event sequence.
```

Comment:
nix the extra space. Applicable everywhere. If you want to move the sentence to new line, you need to put a blank line in between. If you want bullets, you can use markdown style "-"